component/src/main/resources/installation-scripts/install.bat

```diff
@@ -17,19 +17,49 @@ rem specific language governing permissions and limitations
 rem under the License.
 rem
 
-SETLOCAL
-SET /P ballerina_home=Please enter Ballerina home:
+setlocal
 
-IF NOT EXIST "%ballerina_home%/bin/ballerina.bat" (
-    ECHO "Incorrect Ballerina Home provided!"
+IF NOT EXIST "%BALLERINA_HOME%/bin/ballerina.bat" (
+    SET /P ballerina_home=Couldn't find Ballerina home in your system!.Please enter Ballerina home:
```

Comment:
Shall we break this message into 2 lines?

Unable to find Ballerina home in your system!
Please enter Ballerina home: